io.liri.Calculator.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: io.liri.Calculator
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+base: io.liri.BaseApp
+base-version: stable
+command: liri-calculator
+finish-args:
+- --device=dri
+- --filesystem=home
```

Comment:
A calculator should not need this.

Reply:
This one lets you load/save calculations in a script

Reply:
Still shouldn't be needed, the Qt FileChooser dialog should be using the portal to save/load files on the host.

Reply:
Done